### PR TITLE
adding .NET Core 2.2 runtime and updating 2.1 to 2.1.7

### LINKED ISSUE #115: Update ASP.NET Core runtime to 2.2.0
When is ASP.NET Core 2.2 expected to be deployed to GCR?

### PR DESCRIPTION
**Pull request by sharifelgamal**
Fixes #115

### CODE REVIEW

**Comment by henkmollema** [comment]
@sharifelgamal is this still planned to be released this week?

**Comment by sharifelgamal** [comment]
yes, I'm just waiting on a review from @iantalarico to make sure I didn't miss anything major.

**Comment by henkmollema** [comment]
Great, thanks for the heads up!

File: runtimes/versions/aspnetcore-2.2/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .NET Core 2.1 Runtime
```
Comment (iantalarico) [inline runtimes/versions/aspnetcore-2.2/README.md]: nit: 2.2

Comment (sharifelgamal) [reply runtimes/versions/aspnetcore-2.2/README.md]: Fixed.

**Comment by henkmollema** [comment]
Thanks Ian and Sharif!

**Comment by henkmollema** [comment]
@iantalarico @sharifelgamal when will these images be available in GCR?

**Comment by sharifelgamal** [comment]
The release should go out today.

**Comment by henkmollema** [comment]
Thanks again Sharif!

**Pull request merged**